Makefile

```diff
@@ -11,7 +11,7 @@ TEST_LOG_LEVEL ?=
 
 COMMIT = $(shell git rev-parse HEAD)
 SHA = $(shell git rev-parse --short HEAD)
-BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
+BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD | sed 's/\//-/g')
```

Comment:
Not sure if we care about branch naming convention, but I usually have a slash in my branch names.
docker complains about the slashes.